docs/contents/basic_usage.md

```diff
@@ -0,0 +1,102 @@
+# Usage
+
+## Setup
+
+First, `dmenv` needs a Python3 interpreter in PATH, which should be called `python` or `python3`. This should already be the case if you've just installed Python3, regardless of your operating system.
+
+Second, `dmenv` needs a `setup.py` file to work.
+
+* If you don't have a `setup.py` yet, you can run `dmenv init <project name>`
+  to generate one. In this case, make sure to read the comments inside
+  and edit it to fit your needs.
+
+* If you already have one, please note that `dmenv` uses the `extras_require` keyword with a `dev` key
+  to specify development dependencies, which you can use to replace your `dev-requirements.txt`
+  file for instance.
+
+In both cases, here are the contents of `setup.py` file you should end up with:
+
+```python
+from setuptools import setup
+
+setup(
+    name="demo",
+    version="0.6.1",
+    ...
+    install_requires=[
+        "path.py",
+    ],
+    extras_require={
+        "dev": [
+            "pytest",
+        ]
+    },
+
```

Comment:
Silly nitpick, but I notice you omit unnecessary commas above, so it seems to me you can remove the last one here too (or add a "..." here?)